### ISSUE TITLE
µMatrix breaks icloud.com

### ISSUE THREAD

**Issue by aredridel** [uBlockOrigin/uMatrix-issues#191]
### Prerequisites

- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
    - [x] I also searched the existing issues at <https://github.com/gorhill/uMatrix/issues>
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uMatrix](https://old.reddit.com/r/uMatrix/)
- I tried to reproduce the issue when...
    - [x] uMatrix is the only extension
    - [x] uMatrix with default lists/settings
    - [x] using a new, unmodified browser profile
- [x] I am running the latest version of uMatrix
- [x] I checked the [documentation](https://github.com/gorhill/uMatrix/wiki) to understand that the issue I report is not a normal behavior
- [x] I used the logger to rule out that the issue is caused by my ruleset

### Description

µMatrix causes "this content has been blocked by content security policy" warning to show up, even when disabled entirely for that scope.

### A specific URL where the issue occurs

https://www.icloud.com/

### Steps to Reproduce

1. Go to the page https://www.icloud.com/ in Firefox
2. Disable uMatrix for that scope
3. Refresh
4. See error.
5. Disable uMatrix addon entirely
6. Refresh
7. See that error is gone

### Ruleset

Disable uMatrix for the scope.

### Supporting evidence

<img width="1494" alt="Screen Shot 2019-10-13 at 13 58 45" src="https://user-images.githubusercontent.com/2876/66719740-b012bd80-edc1-11e9-8d8a-f6a9ffe1a760.png">

### Your environment

* uMatrix version: 1.4.0
* Browser Name and version: Firefox 69.0.2
* Operating System and version: MacOS 10.15

**Comment by uBlock-user** [comment]
https://github.com/gorhill/uMatrix#warnings

**Comment by uBlock-user** [comment]
Duplicate of https://github.com/uBlockOrigin/uMatrix-issues/issues/130
